### DISCUSSION TITLE
Git Integration

### DISCUSSION THREAD

**Discussion by babakabdolahi** [helix-editor/helix#3186] [General] [answered]
Hi Guys, 
Is there any git integration for helix? The least thing I want is to see the name of my current branch in the status line. Thanks.

**Comment by dam5h** [comment, marked as answer]
See this ticket and the discussion.  https://github.com/helix-editor/helix/issues/227

**Reply by babakabdolahi** [reply]
Thanks, When do you think these features would be available? They are kind of essential features.
